db/payment_icons.yml

```
@@ -946,7 +946,12 @@
   name: satispay
   label: Satispay
   group: wallets
--
+- 
   name: qrph
   label: QR Ph
   group: other
+- 
+  name: arhaus
+  label: arhaus
```

The CI is triggering an error. The label needs to be `Arhaus`
```suggestion
  label: Arhaus
```